benefits/core/templates/core/includes/icon.html

```diff
@@ -9,5 +9,5 @@
   </p>
 {% else %}
   {% load static %}
-  <img class="icon" src="{% static icon.src %}" alt="{{ icon.alt }}" />
+  <img class="icon" width="150px" height="150px" src="{% static icon.src %}" alt="{{ icon.alt }}" />
```

Comment:
Don't need the px here

Reply:
Edited my commit to remove `px` from these two attribute values